api/commands/build.ts

```diff
@@ -85,9 +86,23 @@ export class Builder {
       throw new Error("Protected datasets cannot be fully refreshed.");
     }
 
-    const tasks = table.disabled
-      ? ([] as dataform.IExecutionTask[])
-      : this.adapter.publishTasks(table, this.runConfig, tableMetadata).build();
+    let tasks = table.disabled
+      ? Tasks.create()
+      : this.adapter.publishTasks(table, this.runConfig, tableMetadata);
+
+    let useContextualOps = false;
+    if (this.prunedGraph.projectConfig?.hasOwnProperty("useContextualOps")) {
+      useContextualOps = this.prunedGraph.projectConfig.useContextualOps;
+    }
+    if (table.bigquery?.hasOwnProperty("useContextualOps")) {
+      useContextualOps = table.bigquery.useContextualOps;
+    }
+    if (table.sqlDataWarehouse?.hasOwnProperty("useContextualOps")) {
+      useContextualOps = table.sqlDataWarehouse.useContextualOps;
+    }
+    if (useContextualOps) {
+      tasks = tasks.contextualize();
+    }
```

Comment:
I'm not sure this is the right place to do this. Is adding another parameter to `publishTasks` neater, or even adding the `useContextualOps` option to `runConfig`?